crates/hulks_os_version/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "hulks_os_version"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Please look into the other crate's `Cargo.toml`s. We are trying to...

- sort the keys alphabetically
- add a GPLv3 license

Probably copy-🍝 from another crate and change the name.

Reply:
Or have a look at https://github.com/DevinR528/cargo-sort

Reply:
Alphabetically sorted. Please check if the license name I added is correct